core/silkworm/common/util.cpp

```diff
@@ -24,13 +24,11 @@
 
 namespace silkworm {
 
-ByteView left_pad(ByteView view, size_t min_size) {
+ByteView left_pad(ByteView view, size_t min_size, Bytes& padded) {
```

Comment:
Maybe change name padded -> buffer (to be consistent with signature definition) ?

Reply:
Renamed